homeassistant/components/workday/strings.json

```diff
@@ -31,11 +32,13 @@
     },
     "error": {
       "add_holiday_error": "Incorrect format on date (YYYY-MM-DD)",
-      "remove_holiday_error": "Incorrect format on date (YYYY-MM-DD) or holiday name not found",
-      "already_configured": "[%key:common::config_flow::abort::already_configured_service%]"
+      "remove_holiday_error": "Incorrect format on date (YYYY-MM-DD) or holiday name not found"
     }
   },
   "options": {
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_service%]"
+    },
```

Comment:
Can this actually happen in options since `AbortFlow` is caught?

Reply:
It can't. Thanks for catching my mistake.
I modified the message instead as it's an error and not an abort.